website/source/docs/jobspec/servicediscovery.html.md

```diff
@@ -120,9 +130,14 @@ group "database" {
 * `protocol`: This indicates the protocol for the http checks. Valid options
   are `http` and `https`. We default it to `http`
 
+* `cmd`: This is the command that the Nomad client runs for doing script based
+  health check.
+
+* `args`: Additional arguments to the `cmd` for script based health checks.
+
 ## Assumptions
 
-* Consul 0.6.0 or later is needed for using the TCP checks.
+* Consul 0.6.4 or later is needed for using the TCP checks.
```

Comment:
I think TCP was already working on 0.6:
* Consul 0.6.0 or later is needed for using the TCP checks.
* Consul 0.6.4 or later is needed for using the script checks.